getting-started/troubleshooting.html.md

```diff
@@ -142,4 +142,29 @@ Use these two commands to see environment variables.
 ```
 flyctl secrets list
 flyctl config env
-```
\ No newline at end of file
+```
+
+## _Troubles with WireGuard_
+
+To check if you are connected to the private network you can use the command to get a list of your private networking apps in the same organization.
+```
+dig +short _apps.internal
```

Comment:
Dig probably needs the DNS server specified. Something like `dig +short _apps.internal @fdaa:0:1:3`. This varies per peer, it's included in the wireguard config we generate.

Reply:
There's language about this in the documentation here: 

https://fly.io/docs/reference/privatenetwork/